Sources/FaunaDB/Path.swift

```diff
@@ -27,6 +27,8 @@ extension Path: CustomStringConvertible {
     }
 }
 
+/// Represents a path segment to a field value.
```

Comment:
OK so I think the right thing here is to link to our docs about Paths, but I can't find a spot in our docs that specifically describes how we handle paths. So lets leave it at this and add that, then link to it.